src/Datadog.Trace.ClrProfiler.Managed/Integrations/StackExchange.Redis/RedisBatch.cs

```diff
@@ -23,20 +27,20 @@ public static class RedisBatch
         /// <returns>An asynchronous task.</returns>
         [InterceptMethod(
             Integration = IntegrationName,
-            CallerAssembly = "StackExchange.Redis",
-            TargetAssembly = "StackExchange.Redis",
-            TargetType = "StackExchange.Redis.RedisBase",
-            TargetSignatureTypes = new[] { ClrNames.Ignore, ClrNames.Ignore, ClrNames.Ignore, ClrNames.Ignore },
+            CallerAssembly = RedisAssembly,
+            TargetAssembly = RedisAssembly,
+            TargetType = RedisBaseTypeName,
+            TargetSignatureTypes = new[] { "System.Threading.Tasks.Task`1<T>", "StackExchange.Redis.Message", "StackExchange.Redis.ResultProcessor`1<T>", "StackExchange.Redis.ServerEndPoint" },
             TargetMinimumVersion = Major1,
-            TargetMaximumVersion = Major1)]
+            TargetMaximumVersion = Major2)]
```

Comment:
`ConnectionMultiplexer` has a maximum of Major2 for the same assemblies, this seemed like I missed it the first time around.